stdlib/Random/src/misc.jl

```diff
@@ -326,7 +326,7 @@ function randperm!(r::AbstractRNG, a::Array{<:Integer})
             a[i] = a[j]
         end
         a[j] = i
-        i == 1+mask && (mask = 2mask + 1)
+        i == 1 + mask && (mask = 2 * mask + 1)
```

Comment:
Might want to do a similar reformatting in `randcycle!`

Reply:
Of course. I also added notes to remind people to keep the consistency.